run_unit_tests.sh

```diff
@@ -5,4 +5,4 @@
 
 TEMPDIR=`mktemp -d`
 
-prometheus_multiproc_dir=$TEMPDIR pytest ./tests --cov-report term-missing --cov=. --cov-fail-under=73  --no-cov-on-fail "$@" && prometheus_multiproc_dir=$TEMPDIR python generate_report.py test_reports.toml && rm -rf $TEMPDIR
+prometheus_multiproc_dir=$TEMPDIR pytest  "$@" && prometheus_multiproc_dir=$TEMPDIR python generate_report.py test_reports.toml && rm -rf $TEMPDIR
```

Comment:
Is there a way to still include coverage reports/percentages here like before?

Reply:
Now all this config is inside the `pyproject.toml`

Reply:
now I see it too :-)
it is at https://github.com/RedHatInsights/drift-backend/pull/272/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R26